sdcclient/_monitor.py

```diff
@@ -754,6 +748,52 @@ def get_metrics(self):
         res = requests.get(self.url + '/api/data/metrics', headers=self.hdrs, verify=self.ssl_verify)
         return self._request_result(res)
 
+    def convert_scope_string_to_expression(self, scope):
+        if scope != None:
+            expressions = []
+            string_expressions = scope.strip(' \t\n\r').split(' and ')
+            expression_re = re.compile('^(?P<not>not )?(?P<operand>[^ ]+) (?P<operator>=|!=|in|contains|starts with) (?P<value>.+)$')
+
+            for string_expression in string_expressions:
+                matches = expression_re.match(string_expression)
+
+                if matches == None:
+                    return [False, 'invalid scope format']
+
+                is_not = matches.group('not') != None
+
+                if matches.group('operator') == 'in':
+                    list_value = matches.group('value').strip(' ()')
+                    value_matches = re.findall('[^,]+', list_value)
+
+                    if len(value_matches) == 0:
+                        return [False, 'invalid scope list format']
+
+                    values = map(lambda v: v.strip(' "\''), value_matches)
+                else:
+                    values = [matches.group('value').strip('"\'')]
+
+                if matches.group('operator') == 'in':
+                    operator = 'in' if is_not == False else 'notIn'
+                elif matches.group('operator') == '=':
+                    operator = 'equals'
+                elif matches.group('operator') == '!=':
+                    operator = 'notEquals'
+                elif matches.group('operator') == 'contains':
+                    operator = 'contains' if is_not == False else 'notContains'
```

Comment:
`is_not == False` should be changed to `not is_not` (honestly I would refactor `is_not` name because of this to something more descriptive.

This is happening more times at the PR. Again, if it is to maintain existing code style, I'm OK.